book/content/rdm/rdm.md

```diff
@@ -28,7 +28,7 @@ The following sections in this handbook provide useful context and complementary
 <a name="Summary"></a>
 ## Summary 
 
-Research data management is a key part in encouraging reproducible research. Data management ensures efficiency in research workflows, and also greater reach and impact, as data and software become FAIR (Findable, Accessible, Interoperable and Reuseable). Data should be stored in multiple locations and backed-up regularly. Documentation and metadata ensure that others know how to access, use and re-use your data, and also enable conditions for sharing and publishing data to be outlined. 
+Research data management covers how research data can be stored, described and reused. It is a key part in encouraging reproducible research. Data management ensures efficiency in research workflows, and also greater reach and impact, as data and software become FAIR (Findable, Accessible, Interoperable and Reuseable). Data should be stored in multiple locations and backed-up regularly to prevent loss or data corruption. Clearly describing data using documentation and metadata ensures that others know how to access, use and re-use your data, and also enable conditions for sharing and publishing data to be outlined. 
```

Comment:
:+1: for fixing the content! Would it be okay to make a minor change to the style and have one sentence per line in this section? Thanks!

Reply:
Sure.